### PR TITLE
Unit test coverage for Turbine utils

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Finish enhancing unit test coverage by testing the Turbine utils file

Fixes https://github.com/meroxa/turbine-project/issues/284

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

File: cmd/meroxa/turbine/utils_test.go [hunk 1]
```diff
@@ -52,3 +57,526 @@ func TestCheckGitVersion(t *testing.T) {
 	require.NoError(t, err)
 	assert.False(t, val)
 }
+
+//nolint:funlen,gocyclo
+func TestGitChecks(t *testing.T) {
+	if gh := os.Getenv("GITHUB_WORKSPACE"); gh != "" {
+		t.Skipf("skipping git test in github action")
```
Comment (janelletavares) [inline cmd/meroxa/turbine/utils_test.go:64]: git configuration required to run this test in github

**Comment by simonl2002** [comment]
@janelletavares what's going on with this PR?

**Comment by janelletavares** [comment]
> @janelletavares what's going on with this PR?

@simonl2002 just needs a review

**Review by raulb — APPROVED** [review]
Thank you @janelletavares . This looks great

**Pull request merged**